util/uvmdvgen/checklist.md.tpl

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
@eunchan / @sjgitty 
I copied over this checklist from `doc/project/ip_checklist.md.tpl` so that it gets autogenerated with the testbench. 
Let me know if you think moving it instead of copying it is better.

Reply:
what about symlink? and.. do you have plan to add FPV checklist too?

Reply:
`uvmdvgen` is now vendored into ibex repo. I changed the `doc/project/ip_checklist.md.tpl` to be a symlink to this one.